mod/01_core/set/all/content.rb

```diff
@@ -19,8 +19,11 @@ def raw_content
   structure ? template.db_content : db_content
 end
 
-def chunk_list #override to customize by set
-  :default
+
+format :all do
+  def chunk_list #override to customize by set
+    :default
+  end
```

Comment:
I'm not sure that we will need it, though I can see that it could be useful at some point.

I don't think the change to allow multiple formats is  very costly, and it's all at load time, so it can't hurt.